### ISSUE TITLE
Menu: issues when configuring window.menuBarVisibility: toggle

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#53239]
Steps to Reproduce:

1. configure `window.menuBarVisibility: toggle`
2. press Alt key

=> 🐛 the menu does not appar

3. press Alt+F (for example)

=> 🐛 the menu appears but the title is now gone

![image](https://user-images.githubusercontent.com/900690/42037678-500ef0e4-7ae9-11e8-8eee-96910fbe3bf9.png)

**Comment by sbatten** [comment]
@bpasero, I cannot repro the first issue you mentioned, are you doing this in a VM? you mentioned that alt behavior was strange in your VM. For the title being hidden, this was intended. A recommendation from @kieferrm that looks rather nice imo

**Comment by bpasero** [comment]
@sbatten yes very likely an issue that I run in a VM. And I just remembered how to make it work in Parallels:

![image](https://user-images.githubusercontent.com/900690/42073419-e2cb4aea-7b66-11e8-842a-ac47765739c7.png)

=> Keyboard: Optimize for Games

I think there are still some issues with this approach simply because the behaviour is not what a native menu would do:
* the menu should stay visible and focused when I press `Alt` and not hide again
* the title should not hide, what is the rationale behind that?

**Comment by facebamm** [comment]
Same issue with the 3rd point. I fixed that with a reset the setting property.
